src/adapters/IDB.js

```diff
@@ -81,6 +87,62 @@ export default class IDB extends BaseAdapter {
   }
 
   /**
+   * Batch operations.
+   *
+   * @param  {Function} fn The operation function, which must return a Promise.
+   * @return {Promise}
+   */
+  batch(fn) {
+    return this.open().then(() => {
+      const errors = [], operations = [];
+      const {transaction, store} = this.prepare("readwrite");
+      const batchObject = {
+        get(id) {
+          if (!id) {
```

Comment:
This same check does not seem to be present in https://github.com/Kinto/kinto.js/blob/172-bulk-operations/src/adapters/IDB.js#L203 ?